C/manual/tips-appendix.docbook

```diff
@@ -33,7 +34,23 @@
   <title>&app; Tips and tidbits</title>
 
   <abstract>
-    <para>This chapter gives you some background information about <application>Finance::Quote</application>.
+    <para>This chapter gives you some background information about <application>Finance::Quote</application>.  Due to
+      nature of gathering finance quotes (web scrapping, unstable API) the list of quote sources is constantly evolving.
+    </para>
+
+    <para>Some modules are considered Active, that means it should be possible to fetch quotes using them. Please
+    see <link linkend='fq-active-sources-single'>Active Individual Quote Sources</link>
+    and <link linkend='fq-active-sources-multiple'>Active Multiple Quote Sources</link> for detailed lists.
```

Comment:
like below
```
    see <xref linkend='fq-active-sources-single'/>
    and <xref linkend='fq-active-sources-multiple'/> for detailed lists.
```
is shorter and more fail-safe.

Reply:
done